.github/workflows/ci-scrypto-builder.yml

```diff
@@ -6,6 +6,7 @@ on:
       - develop
       - main
       - release\/*
+  pull_request:
```

Comment:
Not sure if we really want to build docker image upon every PR commit.
Added this just to check if changes in this PR work as expected.

Reply:
Let's leave it as-is. If this causes cost related issues, we will revisit it.